tests/rptest/tests/transactions_test.py

```diff
@@ -737,10 +738,10 @@ def do_upgrade_with_tx(self, selector):
         node_to_upgrade = selector()
 
         # Update node with tx manager
-        self.installer.install(self.redpanda.nodes, RedpandaInstaller.HEAD)
+        self.installer.install(self.redpanda.nodes, (22, 2))
```

Comment:
I don't think this test was specifically about 22.2 -- it should go from HEAD-1 to HEAD.

I'm posting a link in #p_transactions for input from the folks who wrote these tests.

Reply:
i'll wait input for this

Reply:
Yes, looks right (Sorry, for old response)